### DISCUSSION TITLE
Mulitple Locale for search

### DISCUSSION THREAD

**Discussion by alexlakustov** [anpigon/obsidian-book-search-plugin#55] [Ideas]
Hello!

I want to have possibility to set multiple locale for search.
As example, I select English and Polish, and search use English for first request, and Polish for second.

thx.

**Comment by BusterBlade5** [comment]
I wish the plugin had this option. And it seems so trivial to add.

**Comment by MarcoZubieta** [comment]
Urgent feature to add!

**Comment by elknose** [comment]
I love this plugin an I am a multilingual reader and I would love to see this feature realized.

The easiest way to accomplish that could be to put the locale/language as an additional parameter into the search mask such that:

```

Search Book

  : Language like en fr or de :
  
  : Search by keyword or ISBN :
  
```

By that you can avoid mixing up the results of different API-Calls but choose one API-Call upfront.
And hopefully .. we all know which language the book ist written in .. we are looking for at that specific moment. :-)

An YES I would like to see this feature in the near future.
(The workaround by changing the language in the settings is tedious.)

**Comment by geekyorganizer** [comment]
With v0.6.3 (https://github.com/anpigon/obsidian-book-search-plugin/releases) it should be possible to set more than one default language. Does anyone know how to actually do this? I see just the "old" option in the settings and can't figure out, how to set more than one language.
